src/subscription/PaymentMethodInput.js

```diff
@@ -117,7 +117,7 @@ export class PaymentMethodInput {
 				return "creditCardNumberInvalid_msg"
 			} else if (cc.cardHolderName === "") {
 				return "creditCardCardHolderName_msg"
-			} else if (cc.cvv === "") {
+			} else if (cc.cvv === "" || cc.cvv.length !== 3) { // we only support visa, which always has 3-digit CVC (aka CVV2)
```

Comment:
is it visa as in "visa format"? In either case the comment is a bit ambigious

Reply:
I meant we only support paying with VISA credit cards anyway afaik. And VISA (and MasterCard) always uses CVV2 which is 3 digits. American Express also issues 4-digit codes.

But then again, we don't tell the user which cards we support, i think.

Reply:
Well if we support more than Visa why do we say we only support Vista? :D
I think we should allow sending something between 3 and 6 digits, what do you think? Just to make sure we don't lock someone out

Reply:
Hmm? I dont understand. According to (the german) wikipedia the codes are always either 3 or 4 digits. However, only AmEx uses 4 and afaik we dont support them anyways.

Reply:
yes, you are right, we shouldn't hardcode this. fixed

Reply:
Thanks!